OptimoveCore/Classes/Logger/LogLevelCore.swift

```diff
@@ -2,7 +2,7 @@
 
 import Foundation
 
-@objc public enum LogLevel: Int  {
+@objc public enum LogLevelCore: Int  {
```

Comment:
Added a new `LogLevelCore` since Logger moved to the core and to avoid any break in a public API.